server/src/main/scala/com/cloudera/livy/LivyConf.scala

```diff
@@ -90,6 +90,13 @@ object LivyConf {
   val KINIT_FAIL_THRESHOLD =
     LivyConf.Entry("livy.server.launch.kerberos.kinit-fail-threshold", 5)
 
+  // Ldap properties
+  val AUTH_LDAP_URL = Entry("livy.server.auth.ldap.url", null)
+  val AUTH_LDAP_BASE_DN = Entry("livy.server.auth.ldap.base.dn", null)
```

Comment:
I would suggest to change the configuration name to `livy.server.auth.ldap.base-dn`, as well as for the following configurations, would be better to change to `livy.server.auth.ldap.xx-xx-xx`.